PlayStore/fr-FR.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="UTF-8"?><!--
+text.xml file from Click to chat app by TrianguloY.
+Permission given to translate only.
+Instructions for manual translation:
+- Translate the properties making sure not to overpass the maximum character number.
+- Currently this file is copied manually, line breaks are kept and no need to scape special characters.
+If you added/modified the following file add your name here to give credit:
+
+    <!--<string name="translators">Nathanaël Gagnepain (Ilithy)</string>-->
+ -->
+<parameters>
+
+    <!-- Preferably use the name from https://faq.whatsapp.com/general/chats/how-to-use-click-to-chat/ in your language -->
+    <title max_chars="30">Cliquer pour Discuter</title>
+
+    <description_short max_chars="80">Petite application sans publicité pour ouvrir un discussion WhatsApp avec n'importe quel numéro (vous et ce numéro).</description_short>
```

Comment:
This description is unfortunately 116 chars, and Play Store only allows 80 max :(
Could you reduce the size? Maybe use synonyms, but if that's hard just remove some parts of the message (on other translations I had to remove the part between parenthesis for example)

Reply:
My bad, its done